### PR TITLE
incremental work in support of issue #258 and removal of VAX systems

### PR DESCRIPTION
**Pull request by blounsbury-usbr**
Still more to do. I'm planning on removing the PN and Yakima hosts and renaming PNLinux and YakimaLinux to PN and Yakima.

### CODE REVIEW

**Comment by blounsbury-usbr** [comment]
@ktarbet @tjrocha Ok, I think we're good to merge now, or at least from my testing Pisces is working again with all of the HydrometHost options cleared except for PN,GP,Custom while maintain compatibility with databases that have PNLinux, YakimaLinux, or Yakima connection strings.

File: Applications/teacup/Program.cs [hunk 1]
```diff
@@ -27,11 +27,29 @@ static void Main(string[] args)
...
             }
-            //Read the config file
-            string[] lines = File.ReadAllLines(args[2]);
-            if (args[2].Contains("yak"))
-                HServer = HydrometHost.Yakima;
-            
+
+            // try PN database, if down, try Yakima backup
+            double test;
+            try
```
Comment (ktarbet) [inline Applications/teacup/Program.cs:33]: Consider having something Higher Level; so you don't have this type of code in multiple applications. 

for example a  FailoverHydrometSeries(DefaultServer, BackupSever,...) ,  could use both PN and Yakima

File: PiscesTestData/data/ScaleDailyToMonthlyVolumeTest.zip
Comment (ktarbet) [inline PiscesTestData/data/ScaleDailyToMonthlyVolumeTest.zip:1]: Looks like a stray binary file?

File: Applications/teacup/Program.cs (hunk 1 above)
Comment (blounsbury-usbr) [reply Applications/teacup/Program.cs]: Good catch. Actually, this is not needed, deleted by commit 5035c0f558ab05bc02887147b0303259ef7ddc33.

The plan is that if PN hydromet is down, and we need to failover to a PN server placed in Yakima, then the server in Yakima will be configured with the same ip and life goes on...

File: PiscesTestData/data/ScaleDailyToMonthlyVolumeTest.zip
Comment (blounsbury-usbr) [reply PiscesTestData/data/ScaleDailyToMonthlyVolumeTest.zip:1]: It is used by TestScaleDailyToMonthlyVolume. The test unzips the file, which contains a Pisces database and does performs the test. I think Rocha wrote that one, and put it in a zip file to reduce the size of PiscesTestData I presume.

**Review by ktarbet — APPROVED** [review]

**Pull request merged**